### ISSUE TITLE
NTR: Lip: vermilion, lower lip, midline crease or groove (Mouth & throat)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7760]
New term request Lip: vermilion, lower lip, midline crease or groove (elthat_110522181617): Vermillion  UMLS: 1839275

**Comment by pnrobinson** [comment]
@MickeySegal
I am trying to find a reference for this or an associated disease.

Can you confirm you do not mean [Cleft lower lip HP:0010281](https://hpo.jax.org/app/browse/term/HP:0010281)
https://www.sciencedirect.com/science/article/pii/S2214541919300173

**Comment by MickeySegal** [comment]
Also https://omim.org/entry/620107 "Some patients have notching of the upper or lower lip".  But they also have cleft palate, so the cleft lip comment above could apply here too.

**Comment by pnrobinson** [comment]
@sabrinatoro  -- Could we ask your colleages about this term request?

**Comment by sabrinatoro** [comment]
From Mary Marazita (Distinguished Professor at the University of Pittsburg, Department of Oral and Craniofacial Sciences):

This term request is for “midline crease or groove in the lower lip”.   I don’t know what HPO terms there are for lower lips but to answer the comments:

2 Queries: Is this a microform cleft lip?  Is it part of cleft lip?  

The lower lip is not part of orofacial clefts that we have been working on, we have only worked on the upper lip which is different structures and anatomy than the lower lip. I have never heard of a crease in the lower lip seen as a microform (are clefts of the lower lip in the HPO ?) but can’t document this.

**Comment by pnrobinson** [comment]
Looking at Figure 1A of https://pubmed.ncbi.nlm.nih.gov/36084634/, this is a valid feature (see https://www.cell.com/ajhg/fulltext/S0002-9297(22)00361-5). I think it could be defined as

Label: Midline notching of lower lip
Definition: Exaggerated indentation at the very middle (midline) of the lower lip.
Comment: This feature has been observed in Orofaciodigital syndrome XIX (OMIM:620107). See Figure 1a of PMID:36084634.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000569
- label: Midline notching of lower lip
- parents: [Abnormal lower lip morphology (HP:0000178)](https://hpo.jax.org/app/browse/term/HP:0000178)
- pmid: PMID:36084634
- def: Exaggerated indentation at the very middle (midline) of the lower lip.
- comment: This feature has been observed in Orofaciodigital syndrome XIX (OMIM:620107). See Figure 1a of PMID:36084634.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7760
- synonyms:
